src/course/handbook/project-presentations.md

```diff
@@ -0,0 +1,44 @@
+# Project Presentations
+
+Your timetable includes 45 mins for your team to write your presentation and discuss how you are going to present your project together.
+
+**Remember: We want to hear from everyone during your presentation!**
+
+Each team then has 12 minutes to present, with 5 additional minutes for questions. This means each person has about 3 minutes of speaking time. We will be strict with you on timing, so make sure you use your time effectively.
+
+### Structure
+
+1. Show your process:
+   - Talk through the Why, What, How of your project
+   - How did your team split up the work?
+   - How did your team choose to pair? How often did you swap pairs?
+2. Briefly demo your project and explain how it conforms to your brief i.e. the user stories and acceptance criteria
+3. Show your project board and (from Week 2) report on estimated vs actual velocity
+4. Walk us through your code. Emphasise the **things you learnt** that were new to you, which the rest of your cohort could benefit from hearing about:
+   - something you're proud of
+   - something that you found really hard / struggled with
+   - anything you don't understand yet, or don't understand fully
+5. Have the people who didn't write a piece of code talk through it - this is a great way to demonstrate knowledge of learning and collaboration. Your whole team should be able to explain the whole codebase - in other words, everyone is responsible for writing code the whole team understands.
+6. Avoid talking through the project brief for a long time - each team has the same brief and your time will be better spent talking about what was unique about your presentation
+
+### Preparing for presenting
+
+Your team should spend some time preparing what you'll present. Alongside creating a slide deck, this means preparing which parts of the presentation you'll each talk through and ensuring your presentation fits within the time available.
+
+Decide together which sections each person will talk through. Who will share the screen for the demo? Who will talk and who will click through? Choose someone on your team to keep time for the team.
+
+Spend some time revisiting your codebase and understanding the code. Are there sections which you couldn't talk about? If so, spend time reading through the code **together** to ensure you are all on the same page, especially for code you didn't write yourself. Everyone is responsible for this.
+
+### Make it interactive
+
+- **Ask your audience questions** and keep them on their toes.
+- You could even introduce a quiz element to your presentation. Check out [Kahoot!](https://kahoot.com)
```

Comment:
I've never actually seen someone use Kahoot for their Friday presentation - perhaps we can remove this

```suggestion
```

Reply:
Yeah I think the last cohort to do that was ~FAC16 😅